nikola/plugins/command/check.py

```diff
@@ -262,7 +252,7 @@ def analyze(self, task, find_sources=False, check_remote=False):
                         self.logger.warn("Broken link in {0}: {1}".format(filename, target))
                         if find_sources:
                             self.logger.warn("Possible sources:")
-                            self.logger.warn("\n".join(_call_nikola_list(self.l, self.site, ["--deps", task])))
+                            self.logger.warn("\n".join(_call_nikola_list(self.site)))
```

Comment:
The output of `nikola list --all --deps <taskname>` is quite different than the one from `nikola list --all`, so this is not a valid substitution. Actually, I think we need some support from doit here, if we don't want to repeat some of its code or use internal APIs. @schettino72: what do you think (or does that already exist?) about having a simple way in doit to get essentially the output of `doit list --all --deps` (including a list of targets per task) in a nice way?